src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1299,6 +1299,36 @@ private class ACCT_IndividualAccounts_TEST {
         Test.stopTest();
     }
 
+    /*********************************************************************************************************
+    @description 
+        Test Contact insert and update when Household Naming Settings is invalid
+        and the Automatic Household Naming is turned off 
+    verify:
+        The Contact is created and updated successfully
+    **********************************************************************************************************/ 
+    private static testMethod void contactDmlShouldSucceedWhenHHNamingSettingsIsInvalidAndAutomaticHouseholdNamingIsTurnedOff() {
+        turnOffAutomaticHHNaming();
```

Comment:
i'm not used to our code using such long and verbose method names.  we certainly don't do it in normal class methods.  we try to come up with a reasonable sized name that conveys intent, but always provide method headers to provide full details.  why should test methods be any different?